podinfo/config/crd/bases/cilium.io.tetragon.cilium.io_podinfoes.yaml

```diff
@@ -33,19 +33,30 @@ spec:
           metadata:
             type: object
           spec:
-            description: PodInfoSpec defines the desired state of PodInfo. It contains
-              the pod IP and metadata that will be populated by the operator.
+            properties:
+              hostNetwork:
+                description: Host networking requested for this pod. Use the host's
+                  network namespace. If this option is set, the ports that will be
+                  used must be specified.
```

Comment:
Maybe a stupid comment: doc says "the ports that will be used must be specified" but doesn't specify where or how? maybe if it makes sense add it here too?

Reply:
Thank you for pointing that out. I will make the changes.

@tixxdz

Reply:
I haven't mad the changes there yet, will do right before the PR is ready.